setup.cfg

```diff
@@ -5,7 +5,7 @@ ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503, E266
 exclude =
   bin,
   config,
-  doc,
+  doc/conf.py,
```

Comment:
Is this change correct? I tried converting the config in `pipe_tasks`, and I got errors from `doc/_build/html/conf.py` unless I excluded all of `doc`.

Reply:
huh, and I got reports from other devs that conf.py wasn't being ignored if I only listed `doc`! I'll just update `documenteer` to ensure that `conf.py` isn't copied into the build (which I what I should have been doing all along). Thanks for letter me know.

Reply:
It might depend on the flake8 version? I used to get errors from `doc/conf.py` with the old ignore, but now I don't. 😕